api/v1alpha1/dataset_types.go

```diff
@@ -74,10 +74,12 @@ type EncryptOption struct {
 // Mount describes a mounting. <br>
 // Refer to <a href="https://docs.alluxio.io/os/user/stable/en/ufs/S3.html">Alluxio Storage Integrations</a> for more info
 type Mount struct {
-	// MountPoint is the mount point of source.
+	// MountPoint is the mount point of source. <br>
+	// MountPoint can be empty because some runtime don't need to mount external storage (e.g.
+	// <a href="https://v6d.io/">Vineyard</a>).
 	// +kubebuilder:validation:MinLength=5
-	// +required
-	MountPoint string `json:"mountPoint"`
```

Comment:
I think mountPoint is required if `mounts` is defined. I think mounts is no required.

Reply:
Fixed.